### PR TITLE
MNT Fix js lint errors

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/99

I did a merge-up from 2.0 => 2 though there were some `yarn lint` failures which I choose to ignore for the sake of completing the merge-up without having to commit changes without a PR.  I've fixed the `yarn lint` errors in this PR

The `yarn lint` errors were

```
vendor/silverstripe/asset-admin/client/src/containers/AssetAdmin/AssetAdmin.js
  686:13  error  'archiveFiles' is assigned a value but never used  no-unused-vars

vendor/silverstripe/asset-admin/client/src/containers/BulkDeleteConfirmation/BulkDeleteMessage.js
  78:5  error  Prefer fragment shorthand over React.Fragment  react/jsx-fragments

vendor/silverstripe/asset-admin/client/src/containers/BulkDeleteConfirmation/DeletionModal.js
  22:1  error  Expected indentation of 8 spaces but found 10  indent
```

Note this PR is only fixing lint issues, there is a broken behat test on this which will be an existing failure

### CODE REVIEW

**Comment by ChloeSartorelli** [comment]
> there is a broken behat test on this which will be an existing failure

Is there a PR to fix that?

File: client/src/containers/BulkDeleteConfirmation/BulkDeleteMessage.js [hunk 1]
```diff
@@ -75,11 +75,11 @@ const BulkDeleteMessage = ({
     archiveFiles
   );
   return (
-    <Fragment>
+    <>
       <p>{message}</p>
       {(folderDescendantFileTotals.totalItems > 0 || fileTotalItems > 0) &&
       <p>{i18n._t(transKey, transDefault)}</p>}
-    </Fragment>
+    </>
```
Comment (ChloeSartorelli) [inline client/src/containers/BulkDeleteConfirmation/BulkDeleteMessage.js:82]: Also remove it from the import statements.

**Review by ChloeSartorelli — CHANGES_REQUESTED** [review]

File: client/src/containers/BulkDeleteConfirmation/BulkDeleteMessage.js (hunk 1 above)
Comment (emteknetnz) [reply client/src/containers/BulkDeleteConfirmation/BulkDeleteMessage.js:82]: Updated

Comment (emteknetnz) [reply client/src/containers/BulkDeleteConfirmation/BulkDeleteMessage.js:82]: > Is there a PR to fix that?

Not yet

Update - https://github.com/silverstripe/silverstripe-asset-admin/pull/1389

**Review by ChloeSartorelli — APPROVED** [review]
Looks good. I was going to merge up and rerun the behat tests but the auto-merger will fail because of the linting issues and I can't be bothered merging up manually for unrelated behat failures. So, good to go.

**Pull request merged**